lib/dts-lib.robot

```diff
@@ -0,0 +1,130 @@
+*** Settings ***
+Library     ../robot-venv/lib/python3.11/site-packages/robot/libraries/Collections.py
+
+
+*** Keywords ***
+Enable Network Boot
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${dasharo_menu}=    Enter Dasharo System Features    ${setup_menu}
+    ${nw_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Networking Options
+    Set Option State    ${nw_menu}    Enable network boot    ${TRUE}
+    Save Changes And Reset    2    4
+
+Boot Dasharo Tools Suite
+    [Documentation]    Keyword allows to boot Dasharo Tools Suite. Takes the
+    ...    boot method (from USB or from iPXE) as parameter.
+    [Arguments]    ${dts_booting_method}
+    ${boot_menu}=    Enter Boot Menu Tianocore And Return Construction
+    IF    '${dts_booting_method}'=='USB'
+        Enter Submenu From Snapshot    ${boot_menu}    USB SanDisk 3.2Gen1
+    ELSE IF    '${dts_booting_method}'=='USB_emulated'
+        Enter Submenu From Snapshot    ${boot_menu}    PiKVM Composite KVM Device
+        Sleep    1s
+        Press Key N Times    1    ${ENTER}
+    ELSE IF    '${dts_booting_method}'=='iPXE'
+        Enter Submenu From Snapshot    ${boot_menu}    iPXE Network Boot
```

Comment:
Please see `PXE004.001 DTS option is available and works correctly` in `dasharo-compatibility/network-boot.robot`

Reply:
[Replaced](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/c25cb497706d8a74c04839a38cf75f2ad65e1a61#diff-0ca58a364903f744bc9458315dbf5ccc48c37f510ab670103bb67f789341868bR29) to match the approach in `network-boot.robot`